### ISSUE TITLE
Two-digit time zone offsets should be parseable

### ISSUE THREAD

**Issue by mj1856** [moment/moment#1723]
Usually, we see a time zone offset as either `+hhmm` or `+hh:mm`.  However, the ISO 8601 spec allows for the minutes to be omitted for whole-hour offsets.  There is currently not a way to express this is in moment.

See: http://stackoverflow.com/a/24295354/634824

```javascript
var s = "20131107T102103,00+01";

// one should be able to parse:
moment(s,"YYYYMMDD[T]HHmmss[,]SSZZ")

// but instead, zeros have to be added to the input string:
moment(s + "00","YYYYMMDD[T]HHmmss[,]SSZZ")
```

Adding zeros is only a good workaround when you are certain that *every* value will have only two digits.  If some have two and some have four, then by adding zeros, you could end up trying to parse six digits.

**Comment by mj1856** [comment]
No movement yet.  Whenever the fix for this is done, the unit tests removed in #2404 should be added back.

**Comment by janka102** [comment]
Seems to be fixed by this https://github.com/moment/moment/blob/59ba020cf6733f03a9202767517f6edc7af7d706/src/lib/parse/regex.js#L15

Also, #2777 says it's fixed.

**Comment by mj1856** [comment]
Agreed.  This was released in 2.11.0.  The new tests are sufficient as well.

**Comment by mj1856** [comment]
@janka102 - Thanks for finding this.  As you can tell, we could use assistance in issue tracking maintenance. :)
